src/components/Layout/Sections/style.module.less

```diff
@@ -425,7 +425,7 @@
 
 .heroImageOverlay {
   background: @white;
```

Comment:
This is overlaying the image for me, when I deactivate it, I can see the image again.
`.heroImageOverlay { background: @white; }`

Reply:
Ah, alright. We didn't have the illustration yet, when I requested review. :D